intake_demo.ipynb

```diff
@@ -4,43 +4,56 @@
    "cell_type": "code",
```

Comment:
<p>I'm getting the error:</p><pre class="ql-syntax" spellcheck="false">ORCA36 [{'name': 'region', 'description': 'region', 'type': 'str', 'allowed': ['1', '2', '3', '4'], 'default': '1'}, {'name': 'datatype', 'description': 'surface ("surface_hourly"), interior ("interior_daily"), or grid ("grid")', 'type': 'str', 'allowed': ['surface_hourly', 'interior_daily', 'grid'], 'default': 'surface_hourly'}, {'name': 'season', 'description': 'Feb, Mar, Apr ("fma") or Aug, Sep, Oct ("aso")', 'type': 'str', 'allowed': ['fma', 'aso', ''], 'default': ''}, {'name': 'grid', 'description': 'grid type ("bathymetry" or "meshmask")', 'type': 'str', 'allowed': ['bathymetry', 'meshmask', ''], 'default': ''}]
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
&lt;ipython-input-13-ad61998fbeab&gt; in &lt;module&gt;
      1 print(item, params)
----&gt; 2 ds = cat[item](**params).to_dask()
      3 ds

TypeError: ZarrSource object argument after ** must be a mapping, not list
</pre><p>when trying to run this cell...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pangeo-data/swot_adac_ogcms/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
@roxyboy, welcome back! Hope you had a nice vacation.

It looks like you are somehow running a mashup of the version of `intake_demo.ipynb` from the `main` branch of this repo and the version of `intake_demo.ipynb` as proposed by this PR.

The params printed out with your error message above appear to be defined like this
```python
description = cat["ORCA36"].describe()
params = description["user_parameters"]
params
```

<details>

```
[{'name': 'region',
  'description': 'region',
  'type': 'str',
  'allowed': ['1', '2', '3', '4'],
  'default': '1'},
 {'name': 'datatype',
  'description': 'surface ("surface_hourly"), interior ("interior_daily"), or grid ("grid")',
  'type': 'str',
  'allowed': ['surface_hourly', 'interior_daily', 'grid'],
  'default': 'surface_hourly'},
 {'name': 'season',
  'description': 'Feb, Mar, Apr ("fma") or Aug, Sep, Oct ("aso")',
  'type': 'str',
  'allowed': ['fma', 'aso', ''],
  'default': ''},
 {'name': 'grid',
  'description': 'grid type ("bathymetry" or "meshmask")',
  'type': 'str',
  'allowed': ['bathymetry', 'meshmask', ''],
  'default': ''}]
```

</details>

A version of ☝️ this code exists in cell 4 of https://github.com/pangeo-data/swot_adac_ogcms/blob/main/intake_demo.ipynb, but it is not present in the version of the notebook proposed by this PR, here: https://github.com/cisaacstern/swot_adac_ogcms/blob/intake-catalog/intake_demo.ipynb